lib/client.js

```diff
@@ -1463,7 +1463,7 @@ function _resume (client, sync) {
     if (!request.aborted && write(client, request)) {
       client[kPendingIdx]++
     } else {
-      client[kQueue].splice(client[kPendingIdx], 1)
+      util.spliceOneUnordered(client[kQueue], client[kPendingIdx])
```

Comment:
It is necessary to keep the order.
This becomes the last request instead of the aborted request, and the order in which requests are executed is out of order.

Reply:
```suggestion
      client[kQueue].splice(client[kPendingIdx], 1)
```